Dockerfile.server

```diff
@@ -0,0 +1,13 @@
+FROM rust:latest as builder
+WORKDIR /usr/app/src
+COPY ./ /usr/app/src
+RUN apt-get update && apt-get -y install clang && \
+    rm -rf /var/lib/apt/lists/* && \
+    rustup component add rustfmt && \
+    cargo build --lib --bin indradb-server --release
```

Comment:
Running `cargo build` should be a separate step from the above, to better take advantage of caching. Also, you could use buildkit to speed this up:

```
RUN --mount=type=cache,target=/usr/local/cargo/registry \
  --mount=type=cache,target=/rust/target \
  cargo build --release &&\
  cp /rust/target/release/test /rust/test
```

With that, the entire thing won't be recompiled from scratch on every `docker build`.